projects/packages/calypsoify/src/js/mods-gutenberg.js

```diff
@@ -1,45 +1,40 @@
 /* eslint-disable no-var */
-/* global wp, calypsoifyGutenberg */
+/* global calypsoifyGutenberg */
 import '../scss/style-gutenberg.scss';
+import { dispatch, select } from '@wordpress/data';
 
-jQuery( function ( $ ) {
-	// Force fullscreen mode for iframed post editor.
-	if (
-		wp &&
-		wp.data &&
-		wp.data.select( 'core/edit-post' ) &&
-		! wp.data.select( 'core/edit-post' ).isFeatureActive( 'fullscreenMode' )
-	) {
-		wp.data.dispatch( 'core/edit-post' ).toggleFeature( 'fullscreenMode' );
-	}
+// Force fullscreen mode for iframed post editor.
+if (
+	select( 'core/edit-post' ) &&
+	! select( 'core/edit-post' ).isFeatureActive( 'fullscreenMode' )
+) {
+	dispatch( 'core/edit-post' ).toggleFeature( 'fullscreenMode' );
+}
 
-	// Force fullscreen mode for iframed site editor.
-	if (
-		wp &&
-		wp.data &&
-		wp.data.select( 'core/edit-site' ) &&
-		! wp.data.select( 'core/edit-site' ).isFeatureActive( 'fullscreenMode' )
-	) {
-		wp.data.dispatch( 'core/edit-site' ).toggleFeature( 'fullscreenMode' );
-	}
+// Force fullscreen mode for iframed site editor.
+if (
+	select( 'core/edit-site' ) &&
+	! select( 'core/edit-site' ).isFeatureActive( 'fullscreenMode' )
+) {
+	dispatch( 'core/edit-site' ).toggleFeature( 'fullscreenMode' );
+}
 
-	var editPostHeaderInception = setInterval( function () {
-		var $closeButton = $( '.edit-post-fullscreen-mode-close__toolbar a' );
-		if ( $closeButton.length < 1 ) {
-			return;
-		}
-		clearInterval( editPostHeaderInception );
+var editPostHeaderInception = setInterval( function () {
+	var closeButton = document.querySelector( '.edit-post-fullscreen-mode-close__toolbar a' );
+	if ( closeButton.length < 1 ) {
+		return;
+	}
+	clearInterval( editPostHeaderInception );
 
-		if ( calypsoifyGutenberg.closeUrl ) {
-			$closeButton.attr( 'href', calypsoifyGutenberg.closeUrl );
-			$closeButton.attr( 'target', '_parent' );
-		}
-	} );
+	if ( calypsoifyGutenberg.closeUrl ) {
+		closeButton.setAttribute( 'href', calypsoifyGutenberg.closeUrl );
+		closeButton.setAttribute( 'target', '_parent' );
+	}
+} );
 
-	$( 'body.revision-php a' ).each( function () {
-		var href = $( this ).attr( 'href' );
-		if ( href ) {
-			$( this ).attr( 'href', href.replace( '&classic-editor', '' ) );
-		}
-	} );
+$( 'body.revision-php a' ).each( function () {
+	var href = $( this ).getAttribute( 'href' );
+	if ( href ) {
+		$( this ).setAttribute( 'href', href.replace( '&classic-editor', '' ) );
+	}
```

Comment:
Missed the jquery uses down here. Something like this should work.
```suggestion
for ( const node of document.querySelectorAll( 'body.revision-php a' ) ) {
	const href = node.getAttribute( 'href' );
	if ( href ) {
		node.setAttribute( 'href', href.replace( '&classic-editor', '' ) );
	}
```

Reply:
This no longer worked anyway as we needed to check if the page had loaded properly first, so has been rewritten.